src/server/cli/create-server.ts

```diff
@@ -14,15 +13,15 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-
-import * as http from "http";
-
+import { Command } from "commander";
+import { Express } from "express";
+import http from "http";
 import { AddressInfo } from "net";
-import app from "./app";
-import { SERVER_SETTINGS, START_SERVER } from "./config";
+import { ServerSettings } from "../models/server-settings/server-settings";
+
+export default function createServer(serverSettings: ServerSettings, app: Express, program: Command) {
```

Comment:
Again, refactor from module scope variable to function with explicit dependencies. One of the dependencies is commander program reference, so we can error from program and not whole process.